### ISSUE TITLE
FileDropTarget Null Pointer

### ISSUE THREAD

**Issue by bdunn44** [vaadin/framework#9431]
8.1.0.alpha8

The `FileDropTarget` works as expected, however when closing a `Window` containing one a null pointer is thrown:

```
(TypeError) : Cannot read property 'MW' of null
com.google.gwt.core.client.JavaScriptException: (TypeError) : Cannot read property 'MW' of null
   at Unknown.Mhd(com.kbdunn.nimbus.web.NimbusWidgetset-0.js)
   at Unknown.XGc(com.kbdunn.nimbus.web.NimbusWidgetset-0.js)
   at Unknown.S0c(com.kbdunn.nimbus.web.NimbusWidgetset-0.js)
   at Unknown.K0c(com.kbdunn.nimbus.web.NimbusWidgetset-0.js)
   at Unknown.TBc(com.kbdunn.nimbus.web.NimbusWidgetset-0.js)
   at Unknown.g0c(com.kbdunn.nimbus.web.NimbusWidgetset-0.js)
   at Unknown.h0c(com.kbdunn.nimbus.web.NimbusWidgetset-0.js)
   at Unknown.XXc(com.kbdunn.nimbus.web.NimbusWidgetset-0.js)
   at Unknown.eval(com.kbdunn.nimbus.web.NimbusWidgetset-0.js)
   at Unknown.gq(com.kbdunn.nimbus.web.NimbusWidgetset-0.js)
   at Unknown.jq(com.kbdunn.nimbus.web.NimbusWidgetset-0.js)
   at Unknown.eval(com.kbdunn.nimbus.web.NimbusWidgetset-0.js)
   at Unknown._f(http://localhost:8080/VAADIN/vaadinPush.debug.js?v=8.1.0.alpha8)
   at Unknown._invokeFunction(http://localhost:8080/VAADIN/vaadinPush.debug.js?v=8.1.0.alpha8)
   at Unknown._invokeCallback(http://localhost:8080/VAADIN/vaadinPush.debug.js?v=8.1.0.alpha8)
   at Unknown._websocket.onmessage(http://localhost:8080/VAADIN/vaadinPush.debug.js?v=8.1.0.alpha8)
```

This should be reproducible with:

```java
Window popup = new Window();
CssLayout cl = new CssLayout();
new FileDropTarget<>(cl, (e) -> {});
popup.setContent(cl);
getUI().addWindow(popup);
```

**Comment by Artur-** [comment]
Can't reproduce with latest master

**Comment by pleku** [comment]
Duplicate of #9101 and fixed in 8.1.0.beta1 with #9341.
